File: Programmers/LV0/Go/src/두 수의 나눗셈.go
package main

import "fmt"

func solution_두수의나눗셈(num1 int, num2 int) int {
	var result = (float64(num1) / float64(num2)) * 1000
	return int(result)
}

func main() {
	fmt.Println(solution_두수의나눗셈(3, 2))
}

File: Programmers/LV0/Java/두수의나눗셈.java
public class 두수의나눗셈 {
    public static void main(String[] args) {
        Solution_두수의나눗셈 solution = new Solution_두수의나눗셈();
        System.out.println(solution.solution(3, 2));
    }
}

class Solution_두수의나눗셈 {
    public int solution(int num1, int num2) {
        double answer = 0.0;
        answer = (double)num1 / (double)num2;
        answer *= 1000;
        return (int)answer;
    }
}